### PR TITLE
- Adding distribution files instead of build

### PR DESCRIPTION
**Pull request by twtaylorbitgo**
Adds a dist file

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   },
   "repository": {
```
Comment (tylerlevine) [inline package.json:9]: the entry for main above will need to change to `dist/index.js`

Comment (twtaylorbitgo) [reply package.json:9]: sure

Comment (twtaylorbitgo) [reply package.json:9]: wait... i don't think this is right either

**Review by tylerlevine — APPROVED** [review]
lgtm, requiring the babelified version works from node repl

**Pull request merged**